src/fitnesseMain/badCodeToTestCheckstyle.java

```diff
@@ -0,0 +1,53 @@
+package fitnesseMain;
+
+import java.util.*; //BAD: star imports
+
+class  Hello
+{
+  public final static String hello = "helloWorld!"; //BAD: modifier order
+  public String badCommaWhiteSpaceBeforeAndAfter(String input ,String anotherInput)
+    {
+        return "badCommaWhiteSpaceBeforeAndAfter";
+    }
+
+    public void badGenericsWhiteSpace()
+    {
+        List< String> strings = Arrays.asList("hello", "world");
+    }
+
+    public boolean badWhiteSpaces()
+    {
+        int sum = 1+2;
+        System.out. println("hello");
+        return sum==3;
+    }
+
+    @SafeVarargs public final void badAnnotations(String... badVarargsAnnotation)
+    {
+        @SuppressWarnings("default") Hello badSuppressWarningsAnnotation;
+    }
+    
+    public void badArrayTypeStyle()
+    {
+      String args[] = {"hello", "world"};
+    }
+    
+    public void badBraces(){
+      System.out.println("bad");
+      System.out.println("braces!");}
+      
+    public void badMultipleStringLiterals()
+    {
+      //TODO
+      System.out.println("bad multiple string literals");
+    }
+        
+    public void badMultipleEmptyLines()
+    {
+    
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') There is more than 1 empty line one after another. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3ADisallow_Multiple_Empty_Lines)